com.unity.render-pipelines.universal/Shaders/ComplexLit.shader

```diff
@@ -120,6 +120,8 @@ Shader "Universal Render Pipeline/Complex Lit"
             #pragma multi_compile _ _MIXED_LIGHTING_SUBTRACTIVE
             #pragma multi_compile_fragment _ _SCREEN_SPACE_OCCLUSION
             #pragma multi_compile_fragment _ _LIGHT_LAYERS
+            #pragma multi_compile_fragment _ _MAIN_LIGHT_COOKIE
+            #pragma multi_compile_fragment _ _ADDITIONAL_LIGHT_COOKIES
```

Comment:
Just curious, do we have a case where main light support light cookies but addtional light do not?

Reply:
I was wondering the same. Was hoping that these keywords could be merged in to one line.

Reply:
Yes. Directional cookies are supported only for the main light. Point and Spot are supported as additional. If there's only directional light then we don't need to branch on additional lights at all.
A simple use case is fake cloud shadows using cookie on main light for example.

They're also symmetric with other lighting features in URP. I.e. they follow the main/additional split.

I've had some discussions about this and I'll merge these into one to reduce variants. I suppose the separate mainlight isn't that big of a deal (and if it is, we can redo variant for that path).